### ISSUE TITLE
Team mission (clean up trash) bugs

### ISSUE THREAD

**Issue by levimeahan** [elementum-games/shinobi-chronicles#133]
From Gee:
> my team members couldnt do the mission i had to finish it myself and when I finished it it gave no rewards.

https://media.discordapp.net/attachments/883932059893129226/896959774682406912/unknown.png
https://media.discordapp.net/attachments/883932059893129226/896959838809096202/unknown.png

Need to investigate and understand scope of issues, then fix

**Comment by GeeFromEggerth** [comment]
I had this problem when I left the team mission on 6/50 but one of my team member finished it. So I tried to start another one but it says you are already on a solo mission. I go back to the mission tracker by clicking mission it was still saying 6/50 but it wasnt being tracked on the team mission tracker nor could my teammates start  another one. (Not sure if team members can even start a new one themselves) So I finished it and the reward screen popped up but when I went back to check the points I didn't get the 2 points.
